lib/ronin/recon/config.rb

```diff
@@ -366,6 +366,20 @@ def eql?(other)
 
       alias == eql?
 
+      #
+      # Converts config into YAML string.
+      #
+      # @return [String]
+      #
+      def to_yaml
+        res = {}
+
+        res[:params]      = params unless params.empty?
+        res[:concurrency] = concurrency unless concurrency.empty?
+        res[:workers]     = workers.ids.to_a if workers
```

Comment:
I would add a `as_yaml` method to `Workers` so we don't have to reach into the `Workers` class internals.